deployment/upgradeLive

```diff
@@ -1,18 +1,42 @@
 #!/bin/bash
-# For upgrade just type ./upgradeLive <version>
-# This uploads NVM
+# For upgrade just type ./upgradeLive <version> [replace_script|downgrade|build] [ffmpeg build path] [ffmpeg version]
+# in case build is set as parameter then ffmpeg build path and ffmpeg version are mandatory
+
 #set -e
-USAGE="usage: upgradeLive [version] <replace_script|downgrade>"
-[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" || (echo "nvm not found in $NVM_DIR, this is a must, Exiting!" ; exit 1 )
+
+PRODUCT_PATH=/opt/kaltura/liveController
+USAGE="usage: upgradeLive [version] <replace_script|downgrade|build|install_live_recorder> <ffmpeg build path> <ffmpeg version>"
+BUILD_USAGE="usage: upgradeLive [version] build [ffmpeg build path] [ffmpeg version]"
+FFMPEG_BUILD_PATH=
+FFMPEG_VERSION=
+
+if [ -r "$NVM_DIR/nvm.sh" ]; then
```

Comment:
@jessp01  We used NVM for the node.js installation/upgrades.
we don't install explicitly in production and QA